Makefile

```diff
@@ -40,11 +40,18 @@ test-e2e: GO_TEST_PACKAGES :=./test/e2e/...
 test-e2e: GO_TEST_FLAGS += -v
 test-e2e: test-unit
 
+PKG := ${PWD:${GOPATH}/%=%}
```

Comment:
ihttps://github.com/openshift/library-go/blob/master/alpha-build-machinery/make/lib/golang.mk#L3 ?

Reply:
this was leftover from when I was running the container instead of cloning and building, removed